spec/rspec/mocks/stub_spec.rb

```diff
@@ -154,6 +158,15 @@ def object.value; :original; end
           expect(object.value).to eq :stubbed
         end
 
+        it "handles stubbing prepended methods without originally defined on an object's singleton class" do
```

Comment:
I don't understand what "without originally defined" means.  Do you mean this?

``` ruby
it "handles stubbing prepending methods that were only defined on the prepended module"
```

Reply:
I guess so. Sorry for my English :P Will update after I settled all the tests.